src/store/ChartStore.js

```diff
@@ -275,6 +278,8 @@ class ChartStore {
         this.chartNode = this.rootNode.querySelector('.ciq-chart-area');
         this.chartControlsNode = this.rootNode.querySelector('.cq-chart-controls');
 
+        CIQ.Plotter.prototype.yAxiswidth = () => this.yAxiswidth;
```

Comment:
@shayanbinary it'd be better to rename it to `getYAxixWidth` to make explicit that it's a function.